From 6889736b917eed89c4d775824efd8655400e27e3 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <perex@perex.cz>
Date: Mon, 7 Dec 2020 09:33:30 +0100
Subject: [PATCH 4/9] Revert "ucm2: HDA/acp - add Capture simple mixer element
 to the ACP Mic device"

This reverts commit 3320b1ac804595d6c145c1ad7af85917182a3c98.

This solution does not work. The mute state is shared between
the HDA and AMD ACP in PA, so it may cause the security issue, if
the users do not set the mute manually.

BugLink: https://github.com/alsa-project/alsa-ucm-conf/issues/64
Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 ucm2/HDA-Intel/HiFi-acp.conf | 1 -
 1 file changed, 1 deletion(-)

diff --git a/ucm2/HDA-Intel/HiFi-acp.conf b/ucm2/HDA-Intel/HiFi-acp.conf
index 15eeca527907..123ae15f0406 100644
--- a/ucm2/HDA-Intel/HiFi-acp.conf
+++ b/ucm2/HDA-Intel/HiFi-acp.conf
@@ -4,6 +4,5 @@ SectionDevice."Mic1" {
 	Value {
 		CapturePriority 100
 		CapturePCM "hw:${var:AcpCardId}"
-		CaptureMixerElem "Capture"
 	}
 }
-- 
2.26.2

